examples/src/main/java/io/kubernetes/client/examples/springfactorybean/SpringControllerFactoryBeanExample.java

```diff
@@ -0,0 +1,159 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package io.kubernetes.client.examples.springfactorybean;
+
+import io.kubernetes.client.extended.controller.Controller;
+import io.kubernetes.client.extended.controller.reconciler.Reconciler;
+import io.kubernetes.client.extended.controller.reconciler.Request;
+import io.kubernetes.client.extended.controller.reconciler.Result;
+import io.kubernetes.client.informer.SharedInformer;
+import io.kubernetes.client.informer.SharedInformerFactory;
+import io.kubernetes.client.informer.cache.Lister;
+import io.kubernetes.client.openapi.ApiClient;
+import io.kubernetes.client.openapi.models.V1Node;
+import io.kubernetes.client.openapi.models.V1NodeList;
+import io.kubernetes.client.openapi.models.V1Pod;
+import io.kubernetes.client.openapi.models.V1PodList;
+import io.kubernetes.client.spring.extended.controller.KubernetesInformerConfigurer;
+import io.kubernetes.client.spring.extended.controller.annotation.*;
+import io.kubernetes.client.spring.extended.controller.factory.KubernetesControllerFactory;
+import io.kubernetes.client.util.ClientBuilder;
+import java.io.IOException;
+import java.time.Duration;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.stereotype.Component;
+
+@SpringBootApplication
+public class SpringControllerFactoryBeanExample {
```

Comment:
can you replace the old example with this new one? the two examples are pretty identical so i think we can just overwrite the whole old spring example w/o starting a new package.

Reply:
done,
could you tell me when will be the next release?

Reply:
mm i will defer the answer to @brendandburns, but i suppose we're getting close b/c another new release of kubernetes upstream is comming very soon